### PR TITLE
added nonce for like and set featured media

### PR DESCRIPTION
**Pull request by jigneshnakrani088**
added nonce for like and set featured media

### CODE REVIEW

File: app/assets/js/rtMedia.backbone.js [hunk 1]
```diff
@@ -1087,13 +1087,14 @@ jQuery( document ).ready( function ( $ ) {
...
 		var that = this;
+		var like_nonce = $(this).siblings('#rtm_media_like_nonce').val();
 		$( this ).attr( 'disabled', 'disabled' );
 		var url = $( this ).parent().attr( "action" );
 		$( that ).prepend( "<img class='rtm-like-loading' src='" + rMedia_loading_file + "' style='width:10px' />" );
 		$.ajax( {
 			url: url,
 			type: 'post',
-			data: "json=true",
+			data: "json=true&like_nonce="+like_nonce,
```
Comment (Ritesh-patel) [inline app/assets/js/rtMedia.backbone.js]: Instead of string, pass data as an object.

File: app/assets/js/rtMedia.backbone.js [hunk 2]
```diff
@@ -1119,12 +1120,13 @@ jQuery( document ).ready( function ( $ ) {
...
 		var that = this;
 		$( this ).attr( 'disabled', 'disabled' );
+		var featured_nonce = $(this).siblings('#rtm_media_featured_nonce').val();
 		var url = $( this ).parent().attr( "action" );
 		$( that ).prepend( "<img class='rtm-featured-loading' src='" + rMedia_loading_file + "' />" );
 		$.ajax( {
 			url: url,
 			type: 'post',
-			data: "json=true",
+			data: "json=true&featured_nonce="+featured_nonce,
```
Comment (Ritesh-patel) [inline app/assets/js/rtMedia.backbone.js]: Instead of string, pass data as an object.

File: app/main/controllers/media/RTMediaFeatured.php [hunk 3]
```diff
@@ -161,7 +162,13 @@ function process(){
 		if ( ! isset( $this->action_query->id ) ){
 			return;
 		}
+		if( ! wp_verify_nonce( $_REQUEST[ "featured_nonce" ], 'rtm_media_featured_nonce'.$this->media->id ) ){
```
Comment (Ritesh-patel) [inline app/main/controllers/media/RTMediaFeatured.php]: Use $_POST instead as it will always post request.

Comment (jigneshnakrani088) [inline app/main/controllers/media/RTMediaFeatured.php]: okay Done.

**Pull request merged**